### PR TITLE
Ticket 6

### PR DESCRIPTION
**Pull request by han-lawrence**

### CODE REVIEW

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:16]: Refactors or changes to old code shouldn't be included in the PR for this ticket, just to avoid confusion 👍

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:136]: As you use .forEach later in the test, we know that comments must implicitly be an array, so line 136 isn't needed anymore.

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:138]: Nice use of jest-sorted!

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:151]: Nice test - this is usually forgotten!

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:163]: This test description does not match what the test is asserting against. I would change this to be more specific

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:174]: Same here - you're testing that a 404 and error message is returned when the article_id is valid but not in the database.

File: controllers/4-get-article-by-id-controller.js
Comment (callumu21) [inline controllers/4-get-article-by-id-controller.js:1]: I wouldn't number your files like this, personally. The file description should just describe the purpose of the file

File: controllers/5-get-comment.controller.js
Comment (callumu21) [inline controllers/5-get-comment.controller.js:4]: Nice use of destructuring

File: controllers/6-get-api-articles-articleId-comments.controller.js
Comment (callumu21) [inline controllers/6-get-api-articles-articleId-comments.controller.js:12]: Really nice code, here!

File: models/6-fetch-api-articles-articleId-comments.model.js
Comment (callumu21) [inline models/6-fetch-api-articles-articleId-comments.model.js:1]: This file looks identical to 'models/7-fetch-api-articles-articleId-comments.model.js'. You only need to separate out model functions that deal with different contexts (e.g. comments, reviews, users) rather than each different ticket 👍

**Review by callumu21 — APPROVED** [review]
Some really nice code, here! Just be careful with your file structure, as it looks like there's a strange system of numbering and separating things out that's creeping in here. I've left some comments on ways to refactor this and what should be removed, so please look at these before merging.

I would also look at tweaking some of your test descriptions, just to make sure they are as detailed as they need to be to describe what you're testing for.

No need to resubmit this afterwards - it should be good to merge 👍

File: __tests__/app.test.js
Comment (han-lawrence) [reply __tests__/app.test.js:16]: Hey Callum - how can I stop that form happening next time? I thought it was deleted! :-)

Comment (callumu21) [reply __tests__/app.test.js:16]: Refactors like this can just be pushed straight to main. Alternatively, make them on the old branch you had, push them up, and merge them from a PR that doesn't wait for approval 👍

**Pull request merged**
